sanic/request.py

```diff
@@ -105,6 +105,18 @@ def args(self):
 
         return self.parsed_args
 
+    @property
```

Comment:
I'm not sure this will speed things up.  URL and method are checked by the router on every request, and version is used by the response.